src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -217,30 +213,36 @@ limitations under the License.
       <field id="3" name="SessionEnergyCharged" type="energy_mwh" min="0"/>
       <field id="4" name="SessionEnergyDischarged" type="energy_mwh" min="0" optional="true" apiMaturity="provisional"/>
     </event>
+
     <event side="server" code="0x02" name="EnergyTransferStarted" priority="info">
       <description>EnergyTransferStarted</description>
       <field id="0" name="SessionID" type="int32u"/>
       <field id="1" name="State" type="StateEnum" min="0x00" max="0x06"/>
-      <field id="2" name="MaximumCurrent" type="amperage_ma" min="0" max="80000"/>
+      <field id="2" name="MaximumCurrent" type="amperage_ma" min="0"/>
+      <field id="3" name="MaximumDischargingCurrent" type="amperage_ma" optional="true" min="0"/>
```

Comment:
On the spec end: why does it use "MaximumDischargingCurrent" in some places and "MaximumDischargeCurrent" in others?

But this PR reflects that faithfully, so that's ok as far as it goes.

Reply:
I've updated this PR with the change to MaximumDischargeCurrent (to match the spec PR #9317)

Reply:
Thanks!